int32 nodeID_Index
int32 mode
# mode:
# 1. get planned_path
# 2. get nodeID_Index
# 3. set nodeID_Index
---
int32 nodeID_Index
int32[] pathPlanned